Cannot find /error/org.apache.felix.framework

Hi,

starting from today, I am getting the following error during compilation:
```
[INFO] --- bnd-export-maven-plugin:4.1.0-SNAPSHOT:export (default) @ my-app ---
[ERROR] Error   : Cannot find /error/org.apache.felix.framework;version=0 Not found in [my-app [org.eclipse.osgi,org.apache.felix.http.jetty,org.apache.chemistry.opencmis.chemistry-opencmis-commons-api,org.apache.felix.configurator,org.eclipse.equinox.event,org.apache.chemistry.opencmis.chemistry-opencmis-client-bindings,org.osgi.service.event,com.my.app.rest-service,org.apache.aries.javax.jax.rs-api,org.apache.chemistry.opencmis.chemistry-opencmis-commons-impl,org.apache.felix.http.servlet-api,tx-control-provider-jdbc-xa,com.sun.xml.bind.jaxb-core,com.sun.xml.bind.jaxb-impl,org.apache.ws.xmlschema.core,org.apache.neethi,org.apache.cxf.cxf-core,org.apache.servicemix.specs.annotation-api-1.3,slf4j.log4j12,woodstox-core-asl,org.eclipse.equinox.log.stream,org.apache.aries.jax.rs.whiteboard,org.apache.chemistry.opencmis.chemistry-opencmis-client-api,ch.qos.logback.classic,org.apache.felix.scr,org.osgi.util.promise,org.apache.chemistry.opencmis.chemistry-opencmis-client-impl,com.my.app.my-app,com.my.app.cmis-repository,stax2-api,org.apache.cxf.cxf-rt-bindings-soap,javax.validation.api,org.apache.cxf.cxf-rt-databinding-jaxb,org.osgi.service.http.whiteboard,org.apache.cxf.cxf-rt-wsdl,org.osgi.service.jaxrs,org.apache.commons.fileupload,ch.qos.logback.core,org.apache.cxf.cxf-rt-frontend-jaxws,org.apache.commons.io,org.osgi.util.converter,tx-control-provider-jpa-xa,org.apache.cxf.cxf-rt-transports-http,org.osgi.util.function,org.apache.cxf.cxf-rt-frontend-simple,org.apache.felix.configadmin,com.my.app.chemistry-wrapper,org.osgi.service.log,log4j,org.osgi.service.metatype,org.eclipse.equinox.metatype,org.apache.cxf.cxf-rt-ws-policy,com.my.app.api,org.apache.aries.jpa.container,org.osgi.service.jdbc,tx-control-service-xa,slf4j.api,org.apache.cxf.cxf-rt-ws-addr,org.apache.aries.jpa.javax.persistence_2.1,org.apache.cxf.cxf-rt-bindings-xml,org.apache.geronimo.specs.geronimo-osgi-locator,org.osgi.util.pushstream r/w=false]
[ERROR] Error   : org.apache.felix.framework;version=0 Not found in [my-app [org.eclipse.osgi,org.apache.felix.http.jetty,org.apache.chemistry.opencmis.chemistry-opencmis-commons-api,org.apache.felix.configurator,org.eclipse.equinox.event,org.apache.chemistry.opencmis.chemistry-opencmis-client-bindings,org.osgi.service.event,com.my.app.rest-service,org.apache.aries.javax.jax.rs-api,org.apache.chemistry.opencmis.chemistry-opencmis-commons-impl,org.apache.felix.http.servlet-api,tx-control-provider-jdbc-xa,com.sun.xml.bind.jaxb-core,com.sun.xml.bind.jaxb-impl,org.apache.ws.xmlschema.core,org.apache.neethi,org.apache.cxf.cxf-core,org.apache.servicemix.specs.annotation-api-1.3,slf4j.log4j12,woodstox-core-asl,org.eclipse.equinox.log.stream,org.apache.aries.jax.rs.whiteboard,org.apache.chemistry.opencmis.chemistry-opencmis-client-api,ch.qos.logback.classic,org.apache.felix.scr,org.osgi.util.promise,org.apache.chemistry.opencmis.chemistry-opencmis-client-impl,com.my.app.my-app,com.my.app.cmis-repository,stax2-api,org.apache.cxf.cxf-rt-bindings-soap,javax.validation.api,org.apache.cxf.cxf-rt-databinding-jaxb,org.osgi.service.http.whiteboard,org.apache.cxf.cxf-rt-wsdl,org.osgi.service.jaxrs,org.apache.commons.fileupload,ch.qos.logback.core,org.apache.cxf.cxf-rt-frontend-jaxws,org.apache.commons.io,org.osgi.util.converter,tx-control-provider-jpa-xa,org.apache.cxf.cxf-rt-transports-http,org.osgi.util.function,org.apache.cxf.cxf-rt-frontend-simple,org.apache.felix.configadmin,com.my.app.chemistry-wrapper,org.osgi.service.log,log4j,org.osgi.service.metatype,org.eclipse.equinox.metatype,org.apache.cxf.cxf-rt-ws-policy,com.my.app.api,org.apache.aries.jpa.container,org.osgi.service.jdbc,tx-control-service-xa,slf4j.api,org.apache.cxf.cxf-rt-ws-addr,org.apache.aries.jpa.javax.persistence_2.1,org.apache.cxf.cxf-rt-bindings-xml,org.apache.geronimo.specs.geronimo-osgi-locator,org.osgi.util.pushstream r/w=false]
[ERROR] Error   : Cannot find /error/org.apache.felix.framework;version=0 Not found in [my-app [org.eclipse.osgi,org.apache.felix.http.jetty,org.apache.chemistry.opencmis.chemistry-opencmis-commons-api,org.apache.felix.configurator,org.eclipse.equinox.event,org.apache.chemistry.opencmis.chemistry-opencmis-client-bindings,org.osgi.service.event,com.my.app.rest-service,org.apache.aries.javax.jax.rs-api,org.apache.chemistry.opencmis.chemistry-opencmis-commons-impl,org.apache.felix.http.servlet-api,tx-control-provider-jdbc-xa,com.sun.xml.bind.jaxb-core,com.sun.xml.bind.jaxb-impl,org.apache.ws.xmlschema.core,org.apache.neethi,org.apache.cxf.cxf-core,org.apache.servicemix.specs.annotation-api-1.3,slf4j.log4j12,woodstox-core-asl,org.eclipse.equinox.log.stream,org.apache.aries.jax.rs.whiteboard,org.apache.chemistry.opencmis.chemistry-opencmis-client-api,ch.qos.logback.classic,org.apache.felix.scr,org.osgi.util.promise,org.apache.chemistry.opencmis.chemistry-opencmis-client-impl,com.my.app.my-app,com.my.app.cmis-repository,stax2-api,org.apache.cxf.cxf-rt-bindings-soap,javax.validation.api,org.apache.cxf.cxf-rt-databinding-jaxb,org.osgi.service.http.whiteboard,org.apache.cxf.cxf-rt-wsdl,org.osgi.service.jaxrs,org.apache.commons.fileupload,ch.qos.logback.core,org.apache.cxf.cxf-rt-frontend-jaxws,org.apache.commons.io,org.osgi.util.converter,tx-control-provider-jpa-xa,org.apache.cxf.cxf-rt-transports-http,org.osgi.util.function,org.apache.cxf.cxf-rt-frontend-simple,org.apache.felix.configadmin,com.my.app.chemistry-wrapper,org.osgi.service.log,log4j,org.osgi.service.metatype,org.eclipse.equinox.metatype,org.apache.cxf.cxf-rt-ws-policy,com.my.app.api,org.apache.aries.jpa.container,org.osgi.service.jdbc,tx-control-service-xa,slf4j.api,org.apache.cxf.cxf-rt-ws-addr,org.apache.aries.jpa.javax.persistence_2.1,org.apache.cxf.cxf-rt-bindings-xml,org.apache.geronimo.specs.geronimo-osgi-locator,org.osgi.util.pushstream r/w=false]
[ERROR] Error   : org.apache.felix.framework;version=0 Not found in [my-app [org.eclipse.osgi,org.apache.felix.http.jetty,org.apache.chemistry.opencmis.chemistry-opencmis-commons-api,org.apache.felix.configurator,org.eclipse.equinox.event,org.apache.chemistry.opencmis.chemistry-opencmis-client-bindings,org.osgi.service.event,com.my.app.rest-service,org.apache.aries.javax.jax.rs-api,org.apache.chemistry.opencmis.chemistry-opencmis-commons-impl,org.apache.felix.http.servlet-api,tx-control-provider-jdbc-xa,com.sun.xml.bind.jaxb-core,com.sun.xml.bind.jaxb-impl,org.apache.ws.xmlschema.core,org.apache.neethi,org.apache.cxf.cxf-core,org.apache.servicemix.specs.annotation-api-1.3,slf4j.log4j12,woodstox-core-asl,org.eclipse.equinox.log.stream,org.apache.aries.jax.rs.whiteboard,org.apache.chemistry.opencmis.chemistry-opencmis-client-api,ch.qos.logback.classic,org.apache.felix.scr,org.osgi.util.promise,org.apache.chemistry.opencmis.chemistry-opencmis-client-impl,com.my.app.my-app,com.my.app.cmis-repository,stax2-api,org.apache.cxf.cxf-rt-bindings-soap,javax.validation.api,org.apache.cxf.cxf-rt-databinding-jaxb,org.osgi.service.http.whiteboard,org.apache.cxf.cxf-rt-wsdl,org.osgi.service.jaxrs,org.apache.commons.fileupload,ch.qos.logback.core,org.apache.cxf.cxf-rt-frontend-jaxws,org.apache.commons.io,org.osgi.util.converter,tx-control-provider-jpa-xa,org.apache.cxf.cxf-rt-transports-http,org.osgi.util.function,org.apache.cxf.cxf-rt-frontend-simple,org.apache.felix.configadmin,com.my.app.chemistry-wrapper,org.osgi.service.log,log4j,org.osgi.service.metatype,org.eclipse.equinox.metatype,org.apache.cxf.cxf-rt-ws-policy,com.my.app.api,org.apache.aries.jpa.container,org.osgi.service.jdbc,tx-control-service-xa,slf4j.api,org.apache.cxf.cxf-rt-ws-addr,org.apache.aries.jpa.javax.persistence_2.1,org.apache.cxf.cxf-rt-bindings-xml,org.apache.geronimo.specs.geronimo-osgi-locator,org.osgi.util.pushstream r/w=false]
```

As this problem occurs on different machines with different code bases, I guess it is a problem with a dependency.
I searched for a missing dependency, but it seems everything is in place.
Can anybody imagine, what's going wrong?

--- a/archetypes/application/src/main/resources/archetype-resources/debug.bndrun
+++ b/archetypes/application/src/main/resources/archetype-resources/debug.bndrun
@@ -4,6 +4,10 @@ test-index: target/test-index.xml;name="${artifactId} Test"
 
 -standalone: ${index},${test-index}
 
+-runproperties: \
+	osgi.console=,\
+	osgi.console.enable.builtin=false
+
 -runrequires.debug: osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole.plugins.ds)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.gogo.shell)',\

--- a/archetypes/application/src/test/resources/projects/check/reference/debug.bndrun
+++ b/archetypes/application/src/test/resources/projects/check/reference/debug.bndrun
@@ -4,6 +4,10 @@ test-index: target/test-index.xml;name="standalone-app-module Test"
 
 -standalone: ${index},${test-index}
 
+-runproperties: \
+	osgi.console=,\
+	osgi.console.enable.builtin=false
+
 -runrequires.debug: osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole.plugins.ds)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.gogo.shell)',\

--- a/archetypes/project/src/main/resources/archetype-resources/__app-artifactId__/debug.bndrun
+++ b/archetypes/project/src/main/resources/archetype-resources/__app-artifactId__/debug.bndrun
@@ -4,6 +4,10 @@ test-index: target/test-index.xml;name="${app-artifactId} Test"
 
 -standalone: ${index},${test-index}
 
+-runproperties: \
+	osgi.console=,\
+	osgi.console.enable.builtin=false
+
 -runrequires.debug: osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole.plugins.ds)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.gogo.shell)',\

--- a/archetypes/project/src/test/resources/projects/basic/reference/app-module/debug.bndrun
+++ b/archetypes/project/src/test/resources/projects/basic/reference/app-module/debug.bndrun
@@ -4,6 +4,10 @@ test-index: target/test-index.xml;name="app-module Test"
 
 -standalone: ${index},${test-index}
 
+-runproperties: \
+	osgi.console=,\
+	osgi.console.enable.builtin=false
+
 -runrequires.debug: osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole.plugins.ds)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.gogo.shell)',\

--- a/examples/microservice/rest-app-jpa/debug.bndrun
+++ b/examples/microservice/rest-app-jpa/debug.bndrun
@@ -4,6 +4,10 @@ test-index: target/test-index.xml;name="rest-app-jpa Test"
 
 -standalone: ${index},${test-index}
 
+-runproperties: \
+	osgi.console=,\
+	osgi.console.enable.builtin=false
+
 -runrequires.debug: osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole.plugins.ds)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.gogo.shell)',\
@@ -13,10 +17,9 @@ test-index: target/test-index.xml;name="rest-app-jpa Test"
 -runbundles: \
 	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
 	ch.qos.logback.core;version='[1.2.3,1.2.4)',\
-	org.apache.aries.javax.annotation-api;version='[0.0.1,0.0.2)',\
-	org.apache.aries.javax.jax.rs-api;version='[0.0.1,0.0.2)',\
-	org.apache.aries.jax.rs.whiteboard;version='[0.0.1,0.0.2)',\
-	org.apache.felix.configadmin;version='[1.9.0,1.9.1)',\
+	org.apache.aries.javax.jax.rs-api;version='[1.0.0,1.0.1)',\
+	org.apache.aries.jax.rs.whiteboard;version='[1.0.0,1.0.1)',\
+	org.apache.felix.configadmin;version='[1.9.2,1.9.3)',\
 	org.apache.felix.http.jetty;version='[4.0.0,4.0.1)',\
 	org.apache.felix.http.servlet-api;version='[1.1.2,1.1.3)',\
 	org.apache.felix.scr;version='[2.1.0,2.1.1)',\
@@ -55,4 +58,5 @@ test-index: target/test-index.xml;name="rest-app-jpa Test"
 	org.hibernate.javax.persistence.hibernate-jpa-2.1-api;version='[1.0.0,1.0.1)',\
 	osgi.cmpn;version='[4.3.1,4.3.2)',\
 	org.apache.commons.fileupload;version='[1.3.3,1.3.4)',\
-	org.apache.commons.io;version='[2.6.0,2.6.1)'
+	org.apache.commons.io;version='[2.6.0,2.6.1)',\
+	org.apache.servicemix.specs.annotation-api-1.3;version='[1.3.0,1.3.1)'

--- a/examples/microservice/rest-app/debug.bndrun
+++ b/examples/microservice/rest-app/debug.bndrun
@@ -4,6 +4,10 @@ test-index: target/test-index.xml;name="rest-app Test"
 
 -standalone: ${index},${test-index}
 
+-runproperties: \
+	osgi.console=,\
+	osgi.console.enable.builtin=false
+
 -runrequires.debug: osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole.plugins.ds)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.gogo.shell)',\
@@ -13,10 +17,9 @@ test-index: target/test-index.xml;name="rest-app Test"
 -runbundles: \
 	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
 	ch.qos.logback.core;version='[1.2.3,1.2.4)',\
-	org.apache.aries.javax.annotation-api;version='[0.0.1,0.0.2)',\
-	org.apache.aries.javax.jax.rs-api;version='[0.0.1,0.0.2)',\
-	org.apache.aries.jax.rs.whiteboard;version='[0.0.1,0.0.2)',\
-	org.apache.felix.configadmin;version='[1.9.0,1.9.1)',\
+	org.apache.aries.javax.jax.rs-api;version='[1.0.0,1.0.1)',\
+	org.apache.aries.jax.rs.whiteboard;version='[1.0.0,1.0.1)',\
+	org.apache.felix.configadmin;version='[1.9.2,1.9.3)',\
 	org.apache.felix.http.jetty;version='[4.0.0,4.0.1)',\
 	org.apache.felix.http.servlet-api;version='[1.1.2,1.1.3)',\
 	org.apache.felix.scr;version='[2.1.0,2.1.1)',\
@@ -43,4 +46,5 @@ test-index: target/test-index.xml;name="rest-app Test"
 	org.apache.servicemix.specs.json-api-1.1;version='[2.9.0,2.9.1)',\
 	org.apache.johnzon.core;version='[1.1.0,1.1.1)',\
 	org.apache.commons.fileupload;version='[1.3.3,1.3.4)',\
-	org.apache.commons.io;version='[2.6.0,2.6.1)'
+	org.apache.commons.io;version='[2.6.0,2.6.1)',\
+	org.apache.servicemix.specs.annotation-api-1.3;version='[1.3.0,1.3.1)'

--- a/examples/quickstart/app/debug.bndrun
+++ b/examples/quickstart/app/debug.bndrun
@@ -4,6 +4,10 @@ test-index: target/test-index.xml;name="app Test"
 
 -standalone: ${index},${test-index}
 
+-runproperties: \
+	osgi.console=,\
+	osgi.console.enable.builtin=false
+
 -runrequires.debug: osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.webconsole.plugins.ds)',\
  osgi.identity;filter:='(osgi.identity=org.apache.felix.gogo.shell)',\
@@ -13,10 +17,9 @@ test-index: target/test-index.xml;name="app Test"
 -runbundles: \
 	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
 	ch.qos.logback.core;version='[1.2.3,1.2.4)',\
-	org.apache.aries.javax.annotation-api;version='[0.0.1,0.0.2)',\
-	org.apache.aries.javax.jax.rs-api;version='[0.0.1,0.0.2)',\
-	org.apache.aries.jax.rs.whiteboard;version='[0.0.1,0.0.2)',\
-	org.apache.felix.configadmin;version='[1.9.0,1.9.1)',\
+	org.apache.aries.javax.jax.rs-api;version='[1.0.0,1.0.1)',\
+	org.apache.aries.jax.rs.whiteboard;version='[1.0.0,1.0.1)',\
+	org.apache.felix.configadmin;version='[1.9.2,1.9.3)',\
 	org.apache.felix.http.servlet-api;version='[1.1.2,1.1.3)',\
 	org.apache.felix.scr;version='[2.1.0,2.1.1)',\
 	org.osgi.enroute.examples.quickstart.rest;version='[0.0.1,0.0.2)',\
@@ -33,4 +36,5 @@ test-index: target/test-index.xml;name="app Test"
 	org.apache.felix.webconsole.plugins.ds;version='[2.0.8,2.0.9)',\
 	org.apache.commons.fileupload;version='[1.3.3,1.3.4)',\
 	org.apache.commons.io;version='[2.6.0,2.6.1)',\
-	osgi.cmpn;version='[4.3.1,4.3.2)'
+	osgi.cmpn;version='[4.3.1,4.3.2)',\
+	org.apache.servicemix.specs.annotation-api-1.3;version='[1.3.0,1.3.1)'

